tsconfigdoc.json

```diff
@@ -12,7 +12,11 @@
         "./packages/*/src"
       ]
     },
-    "lib": ["DOM", "DOM.iterable", "es2019.array"],
+    "lib": [
+      "DOM",
+      "DOM.iterable",
+      "es2019.array"
```

Comment:
Just for my own curiosity, I'm surprised by `es2019.array`, what needed this?

Reply:
I don't know, this was added automatically. Should I get rid of it?

Reply:
Actually it's [already in master](https://github.com/jupyterlab/jupyterlab/blob/a3634e39af2d3868a7156ec884c4fd67ec545aab/tsconfigdoc.json#L15).